creator-node/src/config.js

```diff
@@ -453,6 +453,30 @@ const config = convict({
     env: 'snapbackDevModeEnabled',
     default: false
   },
+  snapbackReconfigEnabled: {
+    doc: 'Enables replica set reconfiguration or not. Currently enabled for dev mode, disabled for prod.',
+    format: 'BooleanCustom',
+    env: 'snapbackReconfigEnabled',
+    default: false
+  },
+  snapbackModuloBase: {
+    doc: 'The modulo base to segment users by on snapback. Will process `1/snapbackModuloBase` users at some snapback interval',
+    format: 'nat',
+    env: 'snapbackModuloBase',
+    default: 24
+  },
+  snapbackDevJobDelay: {
```

Comment:
similar to the other config values, there's no need for this to have separate dev and prod config vars here, instead just a single `snapbackJobDelay` should suffice, with values set differently according to env